spark-cassandra-connector/src/main/java/com/datastax/spark/connector/japi/CassandraJavaUtil.java

```diff
@@ -1710,11 +1710,11 @@ public static ColumnName column(String columnName) {
     }
 
     public static TTL ttl(String columnName) {
```

Comment:
perhaps you should also add the methods for ttl and writetime with specified default in Java